src/components/NFTPortfolio/Item.vue

```diff
@@ -102,6 +102,7 @@ export default {
   methods: {
     fetchInfo() {
       this.lazyFetchNFTClassAggregatedData();
+      this.updateNFTOwners();
```

Comment:
should we await?

Reply:
not sure if it will cause a slow loading when there are a lot of items.

Reply:
If we don't await `lazyFetchNFTClassAggregatedData` first we risk a race condition here ownerCount is overwritten again